deepdoc/parser/mineru_parser.py

```
@@ -349,7 +350,14 @@ def _run_mineru_api(self, input_path: Path, output_dir: Path, options: MinerUPar
             self.logger.info("[MinerU] Api completed successfully.")
             return Path(output_path)
         except requests.RequestException as e:
-            raise RuntimeError(f"[MinerU] api failed with exception {e}")
+            detail = ""
+            resp = getattr(e, "response", None)
+            if resp is not None:
+                try:
+                    detail = f" body={(resp.text or '')[:2000]}"
+                except Exception:
+                    pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)